README-development.md

```diff
@@ -1,3 +1,39 @@
+# Operating System
+
+## Prerequisites
+
+- git
+- coreos-assembler
+- rpmdistro-gitoverlay
+- ostree
+- rpm-ostree 
+- imagefactory-plugins-TinMan
+- qemu-img
+
+
+## Building
+
+- Clone ``openshift/os``
+- Move into the cloned repo
+- Build packages from source repos:
+ - ``rpmdistro-gitoverlay init``
+ - ``rpmdistro-gitoverlay resolve --fetch-all``
+ - ``rpmdistro-gitoverlay build``
+- Make the ostree
+ - ``export REPO=/tmp/repo``
+ - ``ostree init --repo=$REPO --mode=archive --src $REPO/ --dest=$REPO``
+ - ``coreos-assembler --repo=$REPO host.yaml``
+ - ``ostree summary --repo=$REPO --update --dest $REPO/ --src=$REPO``
+- Make the qcow2
+ - ``ostree --repo=repo remote add rhcos --no-gpg-verify ${OSTREE_INSTALL_URL}``
+ - ``ostree --repo=repo pull --mirror --commit-metadata-only rhcos``
+ - ``imagefactory --debug base_image --file-parameter install_script cloud.ks --parameter offline_icicle True rhcos.tdl``
```

Comment:
This is missing two `sed`s for subbing out `OSTREE_INSTALL_URL` and `OSTREE_INSTALL_REF`. Though we can probably hardcode the latter at least now. (It was originally written to be usable for both CentOS and RHEL-derived input).